integrations/argocd/.port/resources/blueprints.json

```diff
@@ -122,15 +110,18 @@
         },
         "destinationServer": {
           "type": "string",
-          "title": "Destination Server"
+          "title": "Destination Server",
+          "description": "The the URL of the target cluster's Kubernetes control plane API"
         },
         "revision": {
           "type": "string",
-          "title": "Revision"
+          "title": "Revision",
+          "description": "Revision contains information about the revision the comparison has been performed to"
         },
         "targetRevision": {
           "type": "string",
-          "title": "Target Revision"
+          "title": "Target Revision",
+          "description": "TargetRevision defines the revision of the source to sync the application to. In case of Git, this can be commit, tag, or branch"
```

Comment:
```suggestion
          "description": "Target Revision defines the revision of the source to sync the application to. In case of Git, this can be commit, tag, or branch"
```